pkg/juicefs/mount/pod_mount.go

```diff
@@ -287,7 +287,7 @@ func (p *PodMount) createOrAddRef(ctx context.Context, jfsSetting *jfsConfig.Jfs
 	r := builder.NewBuilder(jfsSetting)
 	secret := r.NewSecret()
 	key := util.GetReferenceKey(jfsSetting.TargetPath)
-	for i := 0; i < 120; i++ {
+	for {
```

Comment:
Add the sub-timeout here. Avoid invalid waits.

Reply:
Done